src/components/vacation-requests-table/vacation-requests-table-toolbar/vacation-requests-table-toolbar.tsx

```diff
@@ -0,0 +1,144 @@
+import ToolbarDeleteButton from "./toolbar-delete-button";
+import FormToggleButton from "./toolbar-form-toggle-button";
+import ConfirmationHandler from "../../contexts/confirmation-handler";
+import strings from "../../../localization/strings";
+import { getToolbarTitle } from "../../../utils/toolbar-utils";
+
+/**
+ * Component properties
+ */
+interface Props {
+  deleteVacationRequests: (selectedRowIds: GridRowId[], rows: DataGridRow[]) => Promise<void>;
+  createVacationRequest: (vacationData: VacationData) => Promise<void>;
+  updateVacationRequest: (vacationData: VacationData, vacationRequestId: string) => Promise<void>;
+  setFormOpen: (formOpen: boolean) => void;
+  formOpen: boolean;
+  selectedRowIds: GridRowId[];
+  rows: DataGridRow[];
+}
+/**
+ * Table toolbar component
+ *
+ * @param props component properties
+ */
+const TableToolbar = ({
+  deleteVacationRequests,
+  createVacationRequest,
+  updateVacationRequest,
+  setFormOpen,
+  formOpen,
+  selectedRowIds,
+  rows
+}: Props) => {
+  const [toolbarOpen, setToolbarOpen] = useState(false);
+  const [toolbarFormMode, setToolbarFormMode] = useState<ToolbarFormModes>(ToolbarFormModes.NONE);
+  const [confirmationHandlerOpen, setConfirmationHandlerOpen] = useState(false);
+  const [title, setTitle] = useState(strings.tableToolbar.myRequests);
+
+  useEffect(() => {
+    if (toolbarFormMode) {
+      setTitle(getToolbarTitle(toolbarFormMode));
+    }
+  }, [toolbarFormMode]);
+
+  useEffect(() => {
+    if (selectedRowIds) {
+      setToolbarOpen(true);
+    } else {
+      setToolbarOpen(false);
+    }
+  }, [selectedRowIds]);
+
+  /**
+   * Delete vacation requests and statuses
+   */
+  const deleteVacationsData = async () => {
+    await deleteVacationRequests(selectedRowIds, rows);
+  };
+
+  /**
+   * Toolbar grid item component
+   */
+  const ToolbarGridItem = styled(Grid)({
+    padding: "10px"
+  });
+
+  /**
+   * Toolbar grid container component
+   */
+  const ToolbarGridContainer = styled(Grid)({
+    alignContent: "space-around",
+    alignItems: "center"
+  });
+
+  return (
+    <Box>
+      <ConfirmationHandler
+        open={confirmationHandlerOpen}
+        setOpen={setConfirmationHandlerOpen}
+        deleteVacationsData={deleteVacationsData}
+      />
+      {toolbarOpen && !formOpen && selectedRowIds?.length ? (
+        <ToolbarGridContainer container>
+          <ToolbarGridItem item xs={selectedRowIds?.length === 1 ? 6 : 12}>
+            <ToolbarDeleteButton setConfirmationHandlerOpen={setConfirmationHandlerOpen} />
+          </ToolbarGridItem>
+          {selectedRowIds?.length === 1 ? (
+            <ToolbarGridItem item xs={6}>
+              <FormToggleButton
+                title={strings.tableToolbar.edit}
+                ButtonIcon={Edit}
+                value={formOpen}
+                setValue={setFormOpen}
+              />
+            </ToolbarGridItem>
+          ) : null}
```

Comment:
Could this be a conditonal render with && rather than a ternary, this would allow you to remove the : null

Reply:
Using the && operator instead of ternary